src/more-infos/more-info-configurator.html

```diff
@@ -65,12 +71,26 @@
         </p>
 
         <template is='dom-repeat' items='[[stateObj.attributes.fields]]'>
-          <paper-input
-            label='[[item.name]]'
-            name='[[item.id]]'
-            type='[[item.type]]'
-            on-change='fieldChanged'
-          ></paper-input>
+          <template is='dom-if' if='[[fieldIsCheckbox(item.type)]]'>
+            <paper-checkbox
+              label='[[item.name]]'
+              name='[[item.id]]'
+              type='[[item.type]]'
+              checked='[[item.value]]'
+              on-change='checkboxChanged'
+            >
+              [[item.name]]
+            </paper-checkbox>
+          </template>
+          <template is='dom-if' if='[[fieldIsString(item.type)]]'>
```

Comment:
No need to add `fieldIsString`, you can just do `!fieldIsCheckbox`

Reply:
This code pattern paves the way for more types of inputs. like select boxes or radio buttons.

Reply:
I don't even know if it's a good idea to add more types like this. I rather move to a voluptuous schema from which we infer the schema with something like https://github.com/balloob/voluptuous_form

Reply:
I agree a schema based approach would be nice. But I can't really get a hold of Polymer yet.
I tried to move away from the 'on-change -> setField()' approach to a 'form.serialize()' approach. But I couldn't even get that working.